packages/desktop-client/src/components/schedules/ScheduleLink.tsx

```diff
@@ -44,6 +55,14 @@ export function ScheduleLink({
     }
     actions.popModal();
   }
+  async function onCreate() {
+    const firstId1 = [...ids][0];
+    actions.popModal();
+    pushModal('schedule-edit', {
+      id: null,
+      transaction: getTransaction(firstId1),
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏not quite sure why you would need to spread the `ids` to access 0-index.

```suggestion
    actions.popModal();
    pushModal('schedule-edit', {
      id: null,
      transaction: getTransaction(ids[0]),
```

Reply:
True, removed